src/main/java/edu/harvard/iq/dataverse/ingest/tabulardata/impl/plugins/rdata/RDATAFileReader.java

```diff
@@ -451,7 +434,13 @@ public RDATAFileReader(TabularDataFileReaderSpi originator) {
 
     super(originator);
     
-    
+    // These settings have sane defaults in resources/META-INF/microprofile-config.properties,
+    // ready to be overridden by a sysadmin. Every time a file would be read with this file reader,
+    // a new reader will be created, reading from the cached config source settings with minimal overhead.
+    this.RSERVE_HOST = JvmSettings.RSERVE_HOST.lookup();
+    this.RSERVE_PORT = JvmSettings.RSERVE_PORT.lookup(Integer.class);
```

Comment:
When a non-integer value is given, this code here leads to the problem described by @kcondon: 

> throws a server log error but leaves the user on the grayed out "action" screen and if you click on it, shows upload page with no files, so a silent error.

This problem never occured before, as the setting was never actually used (bug in implementation looking for "dataverse.ingest.rserve.port") and always defaulted to "6311".

How do we proceed? Maybe move the request builder initialization code to `init()` and cache it from there? At least that method is expected to throw an `IOException` (which is not the same), but might at least the calling `read()` function is called from places expecting exceptions.

Reply:
I'll be honest, I may not be fully grasping the seriousness of this issue. As in, "don't use a non-integer for this setting" does seem like an adequate solution, no? 
I mean, there are situations where it may not be obvious or intuitive, that a specific setting needs to be of a certain type. But a port, of all things? What else would anyone want to set it to - "six thousand three hundred twelve"? 
There's also the fact that it's fairly hard to imagine a practical situation where anyone would need to run Rserve on a different port.

Reply:
(otherwise, if we want this to be handled cleanly, I'm guessing the setting rserve.port could be made a String, that will be converted to Integer at the time of use; with the non-integer exception properly intercepted with a rollback to the default 6311 - something like that?)

Reply:
Yes and no - the whole thing would need to be moved to something like the `init()` method, as the requests thing is cached in the `static {}` block.

We could of course also catch the exception and only  log it, then make the field read `null` and check for non-`null` in the code using it and throwing an IOException to prevent NPEs.

Reply:
That, the above, would work. But then there's also an option of merging it as is. (this came up very briefly during the standup and we all seemed to be in agreement that we trusted you on how to proceed with handling this. so, up to you).

Reply:
OK solved by defaulting to `6311` also when invalid integer given. Least impact, better UX.